code/modules/client/preferences.dm

```diff
@@ -1103,7 +1109,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 								continue
 							if(gender == FEMALE && S.gender == MALE)
 								continue
-							if( !(species in S.species_allowed))
+							if(!(species in S.species_allowed))

```

Comment:
I need the extra brackets in places like this for example because if I use !species in S.species_allowed it borks hardcore in the sense that instead of only showing intended hairstyles, it'll show pretty much all hairstyles except for (I think) the first species in the list that doesn't match the species that's being searched for or something.

tl;dr: if I have it like if(!species in S.species_allowed) it breaks, if I leave it like this it's good